docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
       - ./build/mongod_secured.conf:/etc/mongod.conf
 
   ferretdb-postgresql:
-    image: ${FERRETDB_IMAGE:-ferretdb-local}
+    image: ${FERRETDB_IMAGE:-ghcr.io/ferretdb/ferretdb:1}
```

Comment:
… but image name for FerretDB v1?

Reply:
Shouldn't `ferretdb-postgresql` service have ferretdb v1 image? `ferretdb2` service is for v2.

Reply:
My point is that it uses `image:`, not `build:`

Reply:
:facepalm: I see what you mean